conjureup/utils.py

```diff
@@ -136,6 +136,15 @@ def check_bridge_exists():
     return False
 
 
+def check_user_in_group(group):
+    """ Checks if a user is associated with `group`
+    """
+    groups = run_script('id').stdout.decode()
```

Comment:
I think we should use `groups` here instead of `id`. `id` returns more than just the groups and could break if a user was named 'lxd' but not in the 'lxd' group or some other kind of screwup